vmdb/app/views/layouts/listnav/_show_list.html.haml

```diff
@@ -1,88 +1,88 @@
 - if !@def_searches.blank? || !@my_searches.blank?
-  %ul#set_default
-    %li
-    - if @edit && @edit[:expression] && @edit[:expression][:selected] && (!@settings[:default_search] || (@settings[:default_search] && @edit[:expression][:selected][:id].to_i != @settings[:default_search][@edit[@expkey][:exp_model].to_s.to_sym].to_i) || (!@settings[:default_search][@edit[@expkey][:exp_model].to_s.to_sym]))
-      - t = _('Set the current filter as my default')
-      = link_to(image_tag('/images/layout/set-default.png', :border => "0", :alt => t),
-        {:action => "save_default_search", :id => @edit[:expression][:selected][:id].to_i},
-        "data-miq_sparkle_on"  => true,
-        "data-miq_sparkle_off" => true,
-        :remote                => true,
-        :title                 => t)
-    - else
-      = image_tag("/images/layout/set-default-dimmed.png",
-        :border => "0",
-        :title => _("Select a filter to set it as my default"))
+  .panel-group#accordion
+    .panel.panel-default
+      .panel-heading
+        %h4.panel-title
+          - unless @def_searches.blank?
+            = link_to(_("Filters"),
+                      "#collapseOne",
+                      "data-toggle" => "collapse", 
+                      "data-parent" => "#accordion")
+            - if @panels["#{@view.db}_def_searches"].nil?
+              - @panels["#{@view.db}_def_searches"] = true
 
-  #accordion
-    .panel
-      - unless @def_searches.blank?
-        .topbar
-          = link_to(_("Filters"),
-            {:action => 'panel_control', :panel => "#{@view.db}_def_searches"},
-            :remote => true)
-        - if @panels["#{@view.db}_def_searches"].nil?
-          - @panels["#{@view.db}_def_searches"] = true
-        = hidden_div_if(@panels["#{@view.db}_def_searches"].nil? || @panels["#{@view.db}_def_searches"] == false, :id => "#{@view.db}_def_searches") do
-          .panecontent
+      #collapseOne.panel-collapse.collapse.in
+        .panel-body
+          %ul.nav.nav-pills.nav-stacked
             - @def_searches.each do |search|
               - if (!@default_search.blank? && @default_search.name == search.name) || (@default_search.blank? && search.id.to_i == 0)
                 - # highlight if default is selected, or first time in
                 - if @edit && @edit[:expression] && ((@edit[:expression][:selected].nil? && @edit[:selected].nil?) || (!@edit[:expression][:selected].nil? && @edit[:expression][:selected][:id] == search.id.to_i && !@edit[:custom_search]))
-                  - bgcolor = "#d5d5d5"
+                  - li_class = "active"
                 - else
-                  - bgcolor = ""
+                  - li_class = ""
               - elsif @edit && @edit[:expression] && ((@edit[:expression][:selected] && @edit[:expression][:selected][:id] == 0 && search.id.to_i == 0) || (@edit[:expression][:selected] && @edit[:expression][:selected][:name] == search.name && !@edit[:custom_search]))
```

Comment:
that programming inside `haml` looks pretty ugly, we shall get this out of the view, but probably as a part of some other PR